src/screens/FeedScreen/elements/Search.js

```diff
@@ -115,7 +115,7 @@ export const HeaderStyles = StyleSheet.create({
     paddingTop: 7,
     paddingBottom: 7,
     borderBottomWidth: 1,
-    borderBottomColor: COLORS.alto
+    borderBottomColor: COLORS.lightgrey
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change is consistent with the others in improving maintainability. Verify that `COLORS.lightgrey` is defined and matches the previous color `COLORS.alto`.
```diff
-    borderBottomColor: COLORS.alto
+    borderBottomColor: COLORS.lightgrey
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->